src/style/Separator/separator.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+interface SeparatorProps {
+  size: 'small' | 'medium' | 'large';
+}
+
+export const Separator = ({ size } : SeparatorProps) => {
```

Comment:
```suggestion
export const Separator = ({ size = 'medium' } : SeparatorProps) => {
```

Esse é um jeito legal de vc deixar o padrão medium, e aí vc não precisa do else ali embaixo.